scripts/install-worker.sh

```diff
@@ -191,6 +191,13 @@ net.bridge.bridge-nf-call-iptables = 1
 net.ipv4.ip_forward = 1
 EOF
 
+###############################################################################
+### Nerdctl setup #############################################################
+###############################################################################
+
+
+sudo yum install -y nerdctl
```

Comment:
I think we should install this. But I want to make sure that this package doesn't declare `containerd` as a dependency; because if we install it before `containerd`, this might not respect the `containerd_version` AMI template variable. I'll kick off the CI and see what the logs look like.

Reply:
Logs look fine, this installs after `containerd`. 👍